From 041c1278076a516ee7231c46939bea224e6af2f7 Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Thu, 30 Mar 2017 16:04:39 +0200
Subject: [PATCH 15/16] ARM64: dts: meson-gxl-p230: add spdif output interface

Add the necessary pin for the spdif audio interface output and enable it

Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxl-s905d-p230.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxl-s905d-p230.dts b/arch/arm64/boot/dts/amlogic/meson-gxl-s905d-p230.dts
index b9a52b716ffc..5d77ca683d68 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxl-s905d-p230.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxl-s905d-p230.dts
@@ -115,3 +115,13 @@
 		compatible = "brcm,bcm4329-fmac";
 	};
 };
+
+&audio {
+	status = "okay";
+};
+
+&aiu_spdif {
+	pinctrl-0 = <&spdif_out_h_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+};
-- 
2.20.1

